source/extensions/omni.isaac.orbit/docs/CHANGELOG.rst

```diff
@@ -1,6 +1,15 @@
 Changelog
 ---------
 
+0.2.9 (2023-04-18)
```

Comment:
Please upgrade version according to semantic versioning: https://semver.org/

1. MAJOR version when you make incompatible API changes
2. MINOR version when you add functionality in a backwards compatible manner
3. PATCH version when you make backwards compatible bug fixes

Reply:
Also update the `extensions/config.toml` accordingly.

Reply:
You need to update the version to 0.3.0 instead of 0.2.9 since your PR is adding a new feature and isn't a bug fix :)

Reply:
Thanks a lot for letting me learn new things and sorry for the late respondence. I changed the the version to 0.3.0 and so the `extension.toml`.